core/src/main/java/com/gentics/mesh/rest/RestAPIVerticle.java

```diff
@@ -261,7 +261,8 @@ private void applyCommonSettings(HttpServerOptions options, HttpServerConfig ser
 		options.setHost(host);
 		options.setCompressionSupported(true);
 		options.setHandle100ContinueAutomatically(true);
-		// options.setLogActivity(true);
+		options.setUseAlpn(serverConfig.isUseAlpn());
+		options.setLogActivity(true);
```

Comment:
Was this activated on purpose? What kind of log will this produce?

Reply:
No, it is a leak. Removing.